cli/tools/serve.rs

```diff
@@ -46,6 +46,20 @@ pub async fn serve(
 
   let worker_factory =
     Arc::new(factory.create_cli_main_worker_factory().await?);
+
+  if serve_flags.open_site {
+    let host: String;
+    if serve_flags.host == "0.0.0.0" || serve_flags.host == "127.0.0.1" {
+      host = "http://127.0.0.1".to_string();
+    } else if serve_flags.host == "localhost" {
+      host = "http://localhost".to_string();
+    } else {
+      host = format!("https://{}", serve_flags.host);
+    }
```

Comment:
Maybe this should use `http` or `https` depending on the port?

Reply:
Discussed with @bartlomieju and we don't have a good way to do this atm, so we'll just default to http.